tools/towncrier.toml

```diff
@@ -3,7 +3,7 @@ directory = ".changelog.d"
 filename = "CHANGELOG.md"
 template = ".towncrier.template.md"
 start_string = "The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)."
-title_format = "\n## {version} [{project_date}]"
+title_format = "\n## [{version}] ({project_date})"
```

Comment:
sure about this? last time I touched the global towncrier.toml I cried.
there are local versions in the subdirs

Reply:
i tried generating `core/CHANGELOG.md` and it looked alright, will try the others

Reply:
just checking. otherwise utACK, go ahead with the PR if this is indeed correct

Reply:
you're right, this breaks the format of the other monorepo subdirectories
f61f0a2659110835b758d69d8d7b6484a68e95a9